docs/bitcoin-core-usage.md

```diff
@@ -0,0 +1,344 @@
+
+## Setup
+
+Clone the modified Bitcoin Core and build it. Clone HWI.
+
+```
+$ git clone https://github.com/achow101/bitcoin.git -b hww
+$ cd bitcoin
+$ ./autogen.sh
+$ ./configure
+$ make
+$ src/bitcoind -daemon -addresstype=legacy -changetype=legacy
+$ cd ..
+$ git clone https://github.com/achow101/HWI.git
+$ python3 setup.py install
+```
+
+We will do most of our work from HWI, so go there
+
+```
+$ cd hwi
+```
+
+Now we need to find our hardware wallet. We do this using:
+
+```
+$ ./hwi.py enumerate
+[{"fingerprint": "8038ecd9", "serial_number": "205A32753042", "type": "coldcard", "path": "0001:0005:00"}]
+```
+
+For this example, we will use the Coldcard. As we can see, the device path is `0003:0001:00`. The fingerprint of the master key is `8038ecd9`. Now that we have the device, we can issue commands to it. So now we want to get some keys and import them into Core.
+We will be fetching keys at the BIP 84 default.
+
+```
+$ ./hwi.py -f 8038ecd9 getkeypool --bech32 --keypool 0 100
+[{"desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/0/*)", "internal": false, "range": [0, 100], "timestamp": "now", "keypool": true}]
+```
+
+We now createa a new Bitcoin Core wallet and import the keys into Bitcoin Core. The output is formatted properly for Bitcoin Core so it can be copy and pasted.
+
+```
+$ ../bitcoin/src/bitcoin-cli createwallet "coldcard" true
+{
+  "name": "coldcard",
+  "warning": ""
+}
+$ ../bitcoin/src/bitcoin-cli -rpcwallet=coldcard importmulti '[{"desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/0/*)", "internal": false, "range": [0, 100], "timestamp": "now", "keypool": true}]
+'
+[
+  {
+    "success": true
+  }
+]
+```
+
+Now we repeat the `getkeypool` and `importmulti` steps but set a `--internal` flag and use the change keypath (`m/44h/0h/0h/1`) in `getkeypool` to generate change keys.
+
+```
+$ ./hwi.py -f 8038ecd9 getkeypool --bech32 --keypool --internal 0 100
+[{"internal": true, "timestamp": "now", "desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/1/*)", "keypool": true, "range": [0, 100]}]
+$ ../bitcoin/src/bitcoin-cli -rpcwallet=coldcard importmulti '[{"internal": true, "timestamp": "now", "desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/1/*)", "keypool": true, "range": [0, 100]}]
+'
+[
+  {
+    "success": true
+  }
+]
+```
+
+
+The Bitcoin Core wallet is now setup to watch a two hundred keys (100 normal, 100 change) from your hardware wallet and you can use it to track your balances and create transactions. The transactions will need to be signed through HWI.
```

Comment:
Perhaps add:

If you're importing a previously used wallet, you need to run `bitcoin-cli rescanblockchain [blockheight_before_wallet_creation]`

Reply:
Added